cnames_active.js

```diff
@@ -1263,6 +1263,7 @@ var cnames_active = {
   "kmeans": "jeff-tian.github.io/k-means",
   "knacss-vue": "arnissolle.github.io/knacss-vue",
   "knc": "chaituknag.github.io",
+  "knuck": "elcharitas.gitbook.io/knuckjs",
```

Comment:
```suggestion
  "knuck": "hosting.gitbook.com", // noCF
```
I believe this is the configuration that has worked for previous gitbook sites.

Reply:
I've added ithe subdomain with "hosting.gitbook.com" as target to JS.ORGs zonefile but get a 404. @elcharitas can you make sure you have configured gitbook to actually use the subdomain.

Reply:
I've changed it to hosting.gitbook.io but it is not working. I think we can't provide support for gitbook any longer...